pyrgg/test.py

```diff
@@ -31,8 +31,6 @@
 True
 >>> is_float(None)
 False
->>> logger(2,2,2,2,2,2,2,2,2,2,2,2)
-[Error] Logger Failed!
```

Comment:
It'll be better if we add a test which make a failure here. (According to Codecov test failure)

Reply:
You are right, but I couldn't find an example.
Do you have any idea?

Reply:
No.
Just file opening exceptions seems to be a try-except block trigger here.